### ISSUE TITLE
[JIP] Leader logs should not get purged during an epoch

### ISSUE THREAD

**Issue by mark-stopka** [input-output-hk/jormungandr#1396]
**Is your feature request related to a problem/context ? Please describe if applicable.**
Issuing `jcli rest v0 leaders logs get -h http://localhost:3100/api` output changes during epoch, it would be beneficial for pool operators to have them not purged, so they can use the history in scripts and evaluate which blocks made it to the chain and which did not.

**Describe the solution you'd like**
Leader log get's purged (replaced) only with the start of a new epoch.

**Additional context**
Add any other context about the feature request here.

**Comment by Willburn** [comment]
Yes its nice to build a history of this data.  Workaround is to store as file during beginning of epoch and update if successful or not but nicer if already built in.

**Comment by mmahut** [comment]
You can fine tune the rotation of the leader logs as per https://input-output-hk.github.io/jormungandr/configuration/leadership.html

**Comment by mark-stopka** [comment]
@mmahut, great, so all that's required is to change the default values to match the epoch length.
